tests/kafka_cluster_manager/cluster_topology_test.py

```diff
@@ -236,6 +238,34 @@ def test_rebalance_replication_groups(self):
             ct.brokers.keys(),
         )
 
+    def test_rebalance_brokers_for_topic_partition_imbalance(self):
```

Comment:
Not able to reply to your comment.
This test builds cluster-toplogy and rebalance all brokers. So I guess its better here?